src/spikeinterface/extractors/neoextractors/plexon2.py

```diff
@@ -27,10 +34,15 @@ class Plexon2RecordingExtractor(NeoBaseRecordingExtractor):
     NeoRawIOClass = "Plexon2RawIO"
     name = "plexon2"
 
-    def __init__(self, file_path, stream_id=None, stream_name=None, all_annotations=False):
+    def __init__(self, file_path, stream_id=None, stream_name=None, use_names_as_ids=True, all_annotations=False):
         neo_kwargs = self.map_to_neo_kwargs(file_path)
         NeoBaseRecordingExtractor.__init__(
-            self, stream_id=stream_id, stream_name=stream_name, all_annotations=all_annotations, **neo_kwargs
+            self,
+            stream_id=stream_id,
+            stream_name=stream_name,
+            all_annotations=all_annotations,
+            use_names_as_ids=use_names_as_ids,
+            **neo_kwargs,
         )
         self._kwargs.update({"file_path": str(file_path)})
```

Comment:
This use_names_as_ids should also go into kwargs no ?
otherwise when dump it we loose the information.
@h-mayorquin can you check this ?

Reply:
They are added under the hood (maybe a bad thing?) by the neo base extractor:

https://github.com/h-mayorquin/spikeinterface/blob/816ceef031353f4ad7a04b672f554d392827bf27/src/spikeinterface/extractors/neoextractors/neobaseextractor.py#L190-L200

Reply:
sorry I forgot that.

Reply:
Yeah, me too. Today I had to re-check this in:
https://github.com/SpikeInterface/spikeinterface/pull/3193